module/applications/actor/character-sheet-2.mjs

```diff
@@ -607,6 +608,12 @@ export default class ActorSheet5eCharacter2 extends ActorSheet5eCharacter {
       }
     });
 
+    // Apply special context menus for items outside inventory elements
+    const featuresElement = html[0].querySelector(`[data-tab="features"] ${this.options.elements.inventory}`);
```

Comment:
Awkward, but I don't see a better way if we want to retain agnosticism for the `InventoryElement` being placed in either an ActorSheet or ItemSheet.